tools/donate-cpu.py

```diff
@@ -228,22 +228,19 @@ def scanPackage(workPath, cppcheckPath, jobs, fast):
     print('Analyze..')
     os.chdir(workPath)
     libraries = ' --library=posix --library=gnu'
-    if hasInclude('temp', ['<gtk/gtk.h>', '<glib.h>', '<glib/']):
-        libraries += ' --library=gtk'
-    if hasInclude('temp', ['<X11/', '<Xm/']):
-        libraries += ' --library=motif'
-    if os.path.exists(cppcheckPath + '/cfg/python.cfg') and hasInclude('temp', ['<Python.h>']):
-        libraries += ' --library=python'
-    if hasInclude('temp', ['<QApplication>', '<QString>', '<QWidget>', '<QtWidgets>', '<QtGui']):
-        libraries += ' --library=qt'
-    if hasInclude('temp', ['<wx/', '"wx/']):
-        libraries += ' --library=wxwidgets'
-    if hasInclude('temp', ['<zlib.h>']):
-        libraries += ' --library=zlib'
-    if os.path.exists(cppcheckPath + '/cfg/boost.cfg') and hasInclude('temp', ['<boost/']):
-        libraries += ' --library=boost'
-    if hasInclude('temp', ['<SDL.h>']):
-        libraries += ' --library=sdl'
+
+    libraryIncludes = {'gtk': ['<gtk/gtk.h>', '<glib.h>', '<glib/'],
```

Comment:
I'd prefer an alphabetical order.
The list will continue to grow in future.

Reply:
Done, or do you want to sort the includes as well?

Reply:
No, that's enough. Thanks!